fdm-app/app/routes/farm.tsx

```
@@ -88,6 +90,17 @@ export default function App() {
         },
     ])
 
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: posthog

Outer scope values aren't valid dependencies because mutating them doesn't re-render the component.


(lint/correctness/useExhaustiveDependencies)